index.html

```diff
@@ -1134,26 +1139,26 @@ <h2>Expiration</h2>
       </section>
 
       <section>
-        <h2>Status</h2>
-
-        <p>Information about the current status of a <a>verifiable credential</a>,
-        such as suspension or revocation,
-        may be provided by adding the <code>credentialStatus</code> <a>property</a>.
-This property
-       comprises the type of credential status information that is being provided
- (sometimes referred to as the credential status scheme), plus the id of the status
+        <h3>Status</h3>
+
+        <p>
+Information about the current status of a <a>verifiable credential</a>, such 
+as whether it is suspended or revoked, can be provided by adding the 
```

Comment:
Note that there are two conformance issues here
1. The Status is optional, so it MAY be provided in a VC
2. The contents of the Status, if present, MUST conform to that specified here
So we need to reword this to ensure that both conformance statements are present. 

Information about the current status of a VC,...., MAY be provided by adding.....

We already have the MUST statement later on when credentialStatus is defined

N.B. This is a recurring problem in this document, as it tries to combine the two conformance statements into one, which leads to confusion.

Reply:
Done. I split the two requirements using bullet points, which seemed an easier way to present it than in para form.

Reply:
Yes fine thanks